### PR TITLE
metrics: add panels showing pessimistic lock queue lengths

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <eke@fastmail.com>

### What is changed and how it works?

Issue Number: ref #14157 

What's Changed:

Add panels showing pessimistic lock queue lengths, numbers of active waiters and keys

image
image

```commit-message
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by sticnarf** [comment]
> image

I'm a bit confused why the unit is `ops/s`

**Comment by ekexium** [comment]
> I'm a bit confused why the unit is `ops/s`

Oh that must be a mistake

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -45543,4 +45727,4 @@
   "title": "Test-Cluster-TiKV-Details",
   "uid": "RDVQiEzZz",
   "version": 1
-}
+}
```
Comment (sticnarf) [inline metrics/grafana/tikv_details.json:45730]: Recommend to keep newline.

**Review by sticnarf — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**